lib/fog/storage/google_json/requests/list_objects.rb

```diff
@@ -0,0 +1,50 @@
+module Fog
```

Comment:
@selmanj 
`# frozen_string_literal: true` is a magic comment, supported for the first time in Ruby 2.3, that tells Ruby that all string literals in the file are implicitly frozen. The comment must be on the first line of the file. It will be the default in Ruby3, so it's a way to test out the transition from 2.3 onwards.

I would avoid this for now as this needs to be added and tested en-masse.

Thanks for pointing this out, I'll disable the cop for now.